### ISSUE TITLE
option -i --ignore-case ignored

### ISSUE THREAD

**Issue by dicktyr** [Genivia/ugrep#102]
no output as expected

```
% echo abc | ugrep '[^a]b'
```

but the following matches unexpectedly

```
% echo abc | ugrep -i '[^a]b'
abc
```

**Comment by genivia-inc** [comment]
Hm, the difference appears to be related to the Unicode pre-processing. Without Unicode matching (option `-U`) this works exactly like grep (note that option `-U` is used with ugrep for compatibility with grep, turning Unicode matching off):
~~~
echo abc | grep -U -i '[^a]b'
~~~
Does not match as should be.

The Unicode matching will be updated to make this consistent.

Thanks for pointing this out!

**Comment by genivia-inc** [comment]
Fixed in v3.1.7.
